index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <script src="https://kit.fontawesome.com/8a30dfc1c1.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+
+      <div class="headline-wrapper">
+        <div class="app-wrap">
+        
+          <header class="app-header">
+            <a href="#" class="button">
+              <i class="fas fa-bars"></i>
+            </a>
+            
+            <h1>My Logo</h1>  
```

Comment:
- [ ] Please kindly add logo inside an `<a>` tag, because logo is supposed to be a link and users to homepage. This will improve usability of your page 👍🏻 
- [ ] Logo should be aligned left and bars should be aligned right:

## Figma:
![image](https://user-images.githubusercontent.com/77038610/143389282-1f530772-a547-4a13-953a-9a6e35ceee3c.png)

## Yours:
![image](https://user-images.githubusercontent.com/77038610/143389255-54e83703-6f17-4519-9440-8ac34a081b05.png)

Please kindly align logo and bars as given on Figma design 🤗